how-it-works/00_General/01_overview-of-the-internet-computer.card.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Architecture of the Internet Computer
+---
+
+![](/img/how-it-works/subnet_architecture.png)
+
+# Architecture of the Internet Computer
+
+The Internet Computer implements the vision of a World Computer –  an open and secure blockchain-based network that can host programs and data in the form of smart contracts and perform computations on smart contracts in a secure and trustworthy way, while users have to trust that only at least two thirds of hosting nodes are honest.
+To achieve this, the technology behind the Internet Computer (IC) had to be reconceived from ground up, learning many lessons from earlier projects that failed to create a World Computer.
+
+Anyone can deploy a canister smart contract, or just canister, on the Internet Computer.
```

Comment:
Is this the first time we use the term canister? Confusing to non-ICPers. Maybe first call it "smart contract", and then say that on the IC it is called a "canister smart contract" or a "canister". I have no idea myself what the difference between a "smart contract", "canister smart contract", and a "canister" is...but that's just me ;-)